cmd/atlas/docv2.go

```diff
@@ -0,0 +1,53 @@
+//go:build ignore
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"os"
+
+	"ariga.io/atlas/cmd/atlas/internal/cmdapi"
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	docs, err := docs(cmdapi.Root, "")
+	if err != nil {
+		panic(err)
+	}
+	for k, v := range docs {
+		md := "```\n" +
+			string(v) + "\n" +
+			"```"
+		if err := os.WriteFile("../../doc/md/generated/_"+k+".mdx", []byte(md), os.ModePerm); err != nil {
```

Comment:
```suggestion
		md := "```\n" + string(v) + "\n" + "```"
		if err := os.WriteFile("../../doc/md/generated/_"+k+".mdx", []byte(md), os.ModePerm); err != nil {
```

If you're concatenating strings, newlines are not needed.